docs/CQ.rst

```diff
@@ -0,0 +1,59 @@
+Create a Contribution Questionnaire
+===================================
+
+All the new dependencies and all the dependencies updates should be submited as a new CQ into the
```

Comment:
I think this would sound better worded: "Any additions or changes made to GeoTrellis' dependencies must be submitted via a Contribution Questionnaire (CR) to ECLIPSE IP Zilla".